### ISSUE TITLE
I can't create an environment from an environment.yml file

### ISSUE THREAD

**Issue by leilaher** [conda/conda#9690]
Hi everyone,

I received this error while creating an environment from an environment.yml file, using the terminal in Linux to create it, based on a .yml file sent by a colleague: 

conda env create -f environment.yml

[x ~]$ conda env create -f tigramite_p.yml
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.

Traceback (most recent call last):
  File "/software/anaconda-2.2.0/bin/conda-env", line 6, in <module>
    sys.exit(main())
  File "/software/anaconda-2.2.0/lib/python2.7/site-packages/conda_env/cli/main.py", line 63, in main
    return args_func(args, parser)
  File "/software/anaconda-2.2.0/lib/python2.7/site-packages/conda/cli/main.py", line 207, in args_func
    args.func(args, p)
  File "/software/anaconda-2.2.0/lib/python2.7/site-packages/conda_env/cli/main_create.py", line 60, in execute
    env = from_file(args.file)
  File "/software/anaconda-2.2.0/lib/python2.7/site-packages/conda_env/env.py", line 54, in from_file
    return Environment(filename=filename, **data)
TypeError: __init__() got an unexpected keyword argument 'prefix'
[x ~]$ ^C

How can I solve this and create the environment?

**Comment by cielsys** [comment]
@leilaher
tldr: Try removing the **prefix:**  field from the bottom of the yml file.

I had this error message as well in an old, different environment (miniconda3.16 on rasberry pi ubuntu-mate). It seems that **data in 
```return Environment(filename=filename, **data)```  
Contains or is a dict of the yml file and that prefix: field is extraneous? I don't understand what the purpose of that field is. In my yml it was
```prefix: /home/cl/apps/anaconda3/envs/pythonGPU```  
which is where the exported env came from and I think can have no bearing on the machine where the env is being imported to. Deleting that line fixed that error... and let me get to the next errors. :)
